string id
marble_mapping/OctomapArray mapDiffs
marble_artifact_detection_msgs/ArtifactImg[] images
int16[] poseGraph
int32 latestPoseGraph
